### DISCUSSION TITLE
space-x-{num} not working

### DISCUSSION THREAD

**Discussion by ryuhangyeong** [tailwindlabs/tailwindcss#10287] [Help] [answered]
Using space-x in tailwind play works, but not in new nuxt3 project. What to look for? (Other tailwind classes are fine)

<img width="886" alt="image" src="https://user-images.githubusercontent.com/27342882/211557034-d787f4fd-6052-4b2a-ae58-db4d7fe85bff.png">

<img width="716" alt="image" src="https://user-images.githubusercontent.com/27342882/211557164-85451450-a6d9-4f04-8d48-5c1fffa6a4c8.png">

**Comment by wongjn** [comment, marked as answer]
Consider checking:
- The `content` file globs in the Tailwind configuration sufficiently cover files where Tailwind classes are used. For example, perhaps check that the file(s) that contains the code that uses `space-x-3` is covered.
- String interpolation/concatenation is not used to build the class name.

**Reply by ryuhangyeong** [reply]
@wongjn thanks

**Comment by aenial51** [comment]
Did you have a fix on this issue? Facing same issue but no response from anyone and couldn't find this topic troubleshooting in google.
[Dunkin University Website
](https://www.dunkin-university.com/)

**Reply by ryuhangyeong** [reply]
@aenial51

@wongjn's answer helps to solve the problem.

Check out the content of the tailwind configuration file!
